pkg/controller/cincinnati/new.go

```diff
@@ -195,6 +198,32 @@ func (k *kubeResources) newPolicyEngineService(instance *cv1alpha1.Cincinnati) *
 	}
 }
 
+func (k *kubeResources) newPolicyEngineRoute(instance *cv1alpha1.Cincinnati) *routev1.Route {
+	name := namePolicyEngineRoute(instance)
+	return &routev1.Route{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: instance.Namespace,
+			Labels: map[string]string{
+				"app": name,
+			},
+		},
+		Spec: routev1.RouteSpec{
+			Port: &routev1.RoutePort{
+				TargetPort: intstr.FromString("policy-engine"),
+			},
+			To: routev1.RouteTargetReference{
+				Kind: "Service",
+				Name: namePolicyEngineService(instance),
+			},
+			TLS: &routev1.TLSConfig{
+				Termination: routev1.TLSTerminationEdge,
+				InsecureEdgeTerminationPolicy: routev1.InsecureEdgeTerminationPolicyAllow,
+			},
```

Comment:
Any thoughts on how we should tls?

Reply:
InsecureEdgeTerminationPolicyAllow is the default termination policy. The default is also used in the Cincinnati ingress route configuration doc: https://github.com/openshift/cincinnati/blob/master/docs/user/running-cincinnati.md

Reply:
With this configuration, we get a cert issued and signed by ingress-operator. So we are good there. Eg. 
```
Server certificate:
* 	subject: CN=*.apps.suachary-mycluster01.devcluster.openshift.com
* 	start date: May 05 18:49:04 2020 GMT
* 	expire date: May 05 18:49:05 2022 GMT
* 	common name: *.apps.suachary-mycluster01.devcluster.openshift.com
* 	issuer: CN=ingress-operator@1588704534
```